### PR TITLE
Minor fixes in scaling docs

### PR DESCRIPTION
**Pull request by aliok**
@kameshsampath 

I just redid the tutorial, found some minor problems, and fixed them.

BTW, the whole tutorial looks much much better than 3-4 months ago. Great work!

### CODE REVIEW

File: modules/ROOT/pages/04-scaling.adoc [hunk 1]
```diff
@@ -115,6 +115,16 @@ For this exercise let us make our service handle only **10** concurrent requests
...
+Navigate to the tutorial chapter's `knative` folder:
+
+[#scaling-navigate-to-scaling]
+[source,bash,subs="attributes+,+macros"]
+----
+cd $TUTORIAL_HOME/{scaling-repo}/knative
+----
+copyToClipboard::scaling-navigate-to-scaling[]
+
+
```
Comment (aliok) [inline modules/ROOT/pages/04-scaling.adoc:127]: This dir switch was missing.

File: modules/ROOT/pages/04-scaling.adoc [hunk 2]
```diff
@@ -181,7 +191,7 @@ Kubernetes::
 [source,bash,subs="attributes+,+macros",linenums]
 ----
 hey -c 50 -z 10s \
-  -H "Host:prime-generator.knativetutorial.example.com" \
+  -host "prime-generator.knativetutorial.example.com" \
```
Comment (aliok) [inline modules/ROOT/pages/04-scaling.adoc:194]: The `hey` version I use (latest one) doesn't like specifying the host with `-H`. I needed to change that header with `-host`.

File: modules/ROOT/pages/04-scaling.adoc [hunk 3]
```diff
@@ -282,7 +292,7 @@ kubectl::
 [#run-scaling-cleanup]
 [source,bash,subs="attributes+,+macros"]
 ----
-kubectl -n {tutorial-namespace} delete services.serving.knative.dev greeter  &&\
+kubectl -n {tutorial-namespace} delete services.serving.knative.dev greeter
```
Comment (aliok) [inline modules/ROOT/pages/04-scaling.adoc:295]: If the first command fails (and it does if you just follow the tutorial flow), deletion of `prime-generator` fails.

I changed this 1-liner with a 2-liner but using an `||` is also good.

**Comment by kameshsampath** [comment]
this is stale PR .. I have added them in latest

**Pull request closed without merging**